### PR TITLE
IO-3 MACRO - recording driver calls as commands v2

### PR DESCRIPTION
**Pull request by AndrzejBilant**

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/drivers/macro/MacroDriverAdapter.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class MacroDriverAdapter implements Job2dDriver {
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/macro/MacroDriverAdapter.java:12]: Not right. This is decorator!

File: src/test/java/edu/kis/powp/jobs2d/events/SelectLoadMacroListener.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    public SelectLoadMacroListener(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        CommandsFeature.getDriverCommandManager().setCurrentCommand(MacroFeature.getMacroDriverAdapter().getDriverCommandList(), "Macro");
+        driverManager.setCurrentDriver(MacroFeature.getMacroDriverAdapter().getDriver());
+        DriverCommand driverCommand = CommandsFeature.getDriverCommandManager().getCurrentCommand();
+        driverCommand.execute(driverManager.getCurrentDriver());
```
Comment (kowallus) [inline src/test/java/edu/kis/powp/jobs2d/events/SelectLoadMacroListener.java:23]: If name changes from 'run' to 'load' then the logic should also be updated. Please remove it and test if it works using "Run command" from menu.

**Review by kowallus — CHANGES_REQUESTED** [review]
Closer, but not enough.

**Comment by kowallus** [comment]
Reopen this PR, after modifications.

**Comment by BartoszKowalczyk98** [comment]
We don't seem to have an option to reopen our PR.

**Pull request closed without merging**